message/pb/message.proto

```diff
@@ -32,6 +33,9 @@ message Message {
   enum BlockPresenceType {
     Have = 0;
     DontHave = 1;
+    Unknown = 3;
+    AuthRequired = 4;
+    TooBig = 5;
```

Comment:
Not needed here, but we really should have this message rather than just killing streams if the message is too big